tooling/miniutil/src/build/ty_conv.rs

```diff
@@ -8,11 +8,8 @@ pub trait TypeConv {
     fn get_type() -> Type;
 
     // Convenience methods, these should not be overridden.
-    fn get_size() -> SizeStrategy {
-        Self::get_type().size::<DefaultTarget>()
-    }
-    fn get_align() -> Align {
-        Self::get_type().align::<DefaultTarget>()
+    fn get_layout() -> LayoutStrategy {
+        Self::get_type().layout::<DefaultTarget>()
```

Comment:
Might be worth keeping the `get_size/align` methods (with a `Self: Sized` bound)